en/software_update/stm32_bootloader.md

```diff
@@ -60,11 +59,6 @@ These instructions are for the [J-Link GDB server](https://www.segger.com/jlink-
 
 #### Run the JLink GDB server
 
-FMUv1:
```

Comment:
@LorenzMeier I removed this section because it applies to FMUv1. Does this mechanism work for later FMU versions (using these or different instructions?) If so, where can I find those?

Is the AeroFC also still supported? If not, then we should remove this whole JLink section.